java/src/test/java/io/process/analytics/tools/bpmn/generator/internal/BPMNDiagramRichBuilderTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.process.analytics.tools.bpmn.generator.internal;
+
+import static io.process.analytics.tools.bpmn.generator.internal.BpmnInOut.defaultBpmnInOut;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.util.List;
+
+import javax.xml.namespace.QName;
+
+import org.junit.jupiter.api.Test;
+
+import io.process.analytics.tools.bpmn.generator.internal.generated.model.BPMNDiagram;
+import io.process.analytics.tools.bpmn.generator.internal.generated.model.BPMNPlane;
+import io.process.analytics.tools.bpmn.generator.internal.generated.model.TDefinitions;
+
+class BPMNDiagramRichBuilderTest {
+
+    @Test
+    public void initializeBPMNDiagram_when_collaboration_exist_in_bpmn_file() {
+        BPMNDiagramRichBuilder builder = newBuildFromBpmnFile("src/test/resources/bpmn/01-startEvent.bpmn.xml");
```

Comment:
ℹ️ I would prefer accessing test resources using classpath access.

Reply:
yes me too, I was short in time :small_airplane: